unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main/Main.cs

```diff
@@ -87,6 +88,10 @@ protected virtual void Start()
 
             // We trigger the Decentraland logic once everything is initialized.
             WebInterface.StartDecentraland();
+
+            //Crazy test read on https://forum.unity.com/threads/webgl-run-in-background-unacceptable-delay-with-networking.435609/
```

Comment:
I think we do overcome this issue by playing an inaudible audio to keep the background task running. Esteban tried to remove it some weeks ago and re-validate its working